### ISSUE TITLE
Staging Review finding: Radio group error message not announced to screen reader users

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#125319]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** VA Form 21-0779
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Step 1 of 5, when the radio group has an error, and the user clicks continue (putting focus on the the radio group) or tabs back to the radio buttons, the error message is not being announced. Because of this, screen reader users may be blocked from proceeding due to not being able to perceive the error state. 

This exists across the radio groups in the flow. 

JAWS (screen reader) output. Note lack of error, or error message, announcement:

```
MainRegion
Relationship to the Veteran group
What is your relationship to the Veteran? (*Required) group
  I’m from a state cemetery radio button not checked
1 of 2
Submit A State Or Tribal Organization Request For Interment Allowance | Veterans Affairs document
To change the selection press Up or Down Arrow.
  I’m from a state cemetery radio button not checked
1 of 2
Back Button
To activate press Enter.
Continue Button
To activate press Enter.
Relationship to the Veteran group
```

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="641" height="343" alt="radio error" src="https://github.com/user-attachments/assets/d6ffcd81-4680-4570-b2da-08343f8134ca" />

</details>

## Recommended action

Ensure that the field is identified as in error progammatically using `aria-invalid="true"` and ensure that the error message is associated to the field in error, so that screen readers announce the error when the radio has focus. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by LinnJS** [comment]
Hey @jasonday thank you so much for all your hard work with the QA feedback has been a huge help.

Just wanted to callout that it seems like this a11y issue might be a long standing issue with the VADS component.

I found this [comment in the code](https://github.com/department-of-veterans-affairs/vets-website/blob/ae0f712b6c3f291de5e2c78c52850243667434f2/src/platform/forms-system/src/js/web-component-fields/VaCheckboxGroupField.jsx#L84) from around 2 years ago that is is a known issue.

Comment reads `{/* known a11y issue: this will not be read out */}` on the [VaCheckboxGroupField.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/ae0f712b6c3f291de5e2c78c52850243667434f2/src/platform/forms-system/src/js/web-component-fields/VaCheckboxGroupField.jsx).

**Comment by jasonday** [comment]
Added additional labels so that the ticket gets added to the DST workflow for review. Please follow up with them. TY!

**Comment by leahkeeler** [comment]
@jasonday I assume this is not launch blocking? We will keep this in our backlog and implement once a solution is identified, is there a label y'll use for known bugs?

**Comment by voidspooks** [comment]
@evoeges (Elyse) is taking on this.

**Comment by evoeges** [comment]
This appears to be resolved. "Error you must provide a response" is being read by the screen reader. 
Tested with VoiceOver on macOS v15.6.

**Comment by tbaker1026** [comment]
@leahkeeler The Design system has a[ PR ](https://github.com/department-of-veterans-affairs/vets-website/pull/39738)out for a fix, hoping to release it this week.

**Comment by tbaker1026** [comment]
@leahkeeler Can you verify and close issue?  Thank you
